_resources/port1.0/group/crossgcc-1.0.tcl

```diff
@@ -70,10 +76,17 @@ proc crossgcc.setup {target version} {
 
         homepage        http://gcc.gnu.org/
         master_sites    gnu:gcc/gcc-${version}/:gcc
-        use_bzip2       yes
+
+        # TODO: cheating
+        # the xz is used for 6.4.0, 7.2.0 and probably later
+        if {[lindex [split ${crossgcc.version} .] 0] < 7} {
```

Comment:
I would not say this is cheating, it is a valid assumption. You could also use `vercmp` for that:
```
if {[vercmp ${crossgcc.version} 7.0] < 0} {
    ...
}
```

Reply:
It is cheating in the sense that 7.1 and 6.3 use tar.bz2 while 7.2 and 6.4 uses .tar.xz. I wasn't sure how to compare the whole version, but yes, `vercmp` makes a lot more sense, thank you very much.